packages/mdc-side-sheet/README.md

```diff
@@ -1,4 +1,318 @@
-# Side Sheet
+<!--docs:
+title: "Side Sheets"
+layout: detail
+section: components
+excerpt: "Permanent, dismissible, and modal side sheets."
+iconId: side_navigation
+path: /catalog/side-sheets/
+-->
 
-The [Side Sheet component](https://material.io/go/design-sheets-side) is yet to be completed, please follow the [tracking issue](https://github.com/material-components/material-components-web/issues/2665) for more information.
+# Side Sheets
 
+The MDC Side Sheet is used to organize access to destinations and other functionality on an app.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/go/design-navigation-side-sheet">Material Design guidelines: Navigation side-sheet</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components.github.io/material-components-web-catalog/#/component/side-sheet">Demo</a>
+  </li>
+</ul>
+
+## Installation
+
+```
+npm install @material/side-sheet
+```
+
+### HTML Structure
+
+```html
+<aside class="mdc-side-sheet">
+  <div class="mdc-side-sheet__content">
+    <div><label><input type="checkbox"> Events</label></div>
```

Comment:
Leave the inner contents out, just put ..., since we have no opinion on the internals (but what's here now isn't MD-ified at all).